pandas/core/dtypes/common.py

```diff
@@ -120,45 +120,52 @@ def is_object_dtype(arr_or_dtype):
 
 
 def is_sparse(arr):
-    """Check whether an array-like is a pandas sparse array.
+    """
+    Check whether an array-like is a 1-D pandas sparse array.
 
     Check that the one-dimensional array-like is a pandas sparse array.
     Returns True if it is a pandas sparse array, not another type of
     sparse array.
```

Comment:
Personally I find this paragraph not so clear. I think something like this would be shorter and also clearer: "Return `True` if `arr` is `pandas.SparseArray` or `pandas.SparseSeries`, and `False` for any other type."